metrics/opentelemetry-metrics/src/main/scala/sttp/tapir/server/metrics/opentelemetry/OpenTelemetryMetrics.scala

```diff
@@ -36,6 +36,33 @@ case class OpenTelemetryMetrics[F[_]](meter: Meter, metrics: List[Metric[F, _]])
 
 object OpenTelemetryMetrics {
 
+  /** Default labels for OpenTelemetry-compliant metrics, as recommended here:
+    * https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#http-server
+    *
+    *   - `http.request.method` - HTTP request method (e.g., GET, POST).
+    *   - `path` - The request path or route template.
+    *   - `http.response.status_code` - HTTP response status code (200, 404, etc.).
+    */
+  lazy val OpenTelemetryAttributes: MetricLabels = MetricLabels(
+    forRequest = List(
+      "http.request.method" -> { case (_, req) => req.method.method },
+      "url.scheme" -> { case (_, req) => req.uri.scheme.getOrElse("unknown") },
+      "path" -> { case (ep, _) => ep.showPathTemplate(showQueryParam = None) }
+    ),
+    forResponse = List(
+      "http.response.status_code" -> {
+        case Right(r) => r.code.code.toString
+        // Default to 500 for exceptions
+        case Left(_) => "500"
+      },
+      "error.type" -> {
+        case Left(ex) => ex.getClass.getSimpleName
+        case Right(_) => ""
```

Comment:
```suggestion
    forResponse = List(
      "http.response.status_code" -> {
        case Right(r) => r.code.code.toString
        // Default to 500 for exceptions
        case Left(_) => "500"
      },
      "error.type" -> {
        case Left(ex) => ex.getClass.getSimpleName
        case Right(_) => ""
```
  forResponse = List(
    "http.response.status_code" -> {
      case Right(r) => r.code.code.toString 
      case Left(_) => "500" 
    },
    "error.type" -> {
      case Left(ex) => Some(ex.getClass.getName) 
      case Right(_) => None 
    }
  ).collect { case (k, Some(v)) => k -> v }

Reply:
the `http.response.status_code` would also have to return an `Option`, so the results would have to be wrapped in a `Some`